data-reporting-service/observation-service/src/main/resources/application.yaml

```diff
@@ -0,0 +1,33 @@
+spring:
+  kafka:
+    stream:
+      input:
+        observation:
+          topic-name: cdc.nbs_odse.dbo.Observation
+        defaultData:
+          topic-name: cdc.nbs_odse.dbo.Default.input
+      output:
+          observation:
+            topic-name: cdc.nbs_odse.dbo.Observation.output
+            topic-name-transformed: cdc.nbs_odse.dbo.Observation.output-transformed
```

Comment:
If this is going to be consumed by Kafka-Connector, it should be of the form `nbs_observations`

Reply:
This one will be fixed with the other as a part of optimization